src/sentry/features/handler.py

```diff
@@ -28,13 +43,13 @@ def has_for_batch(self, batch):
 
 
 class BatchFeatureHandler(FeatureHandler):
-    def _check_for_batch(self, feature_name, organization, actor):
+    def _check_for_batch(self, feature_name: str, organization: Any, actor: Any) -> bool:
         raise NotImplementedError
 
-    def has(self, feature, actor):
-        organization = getattr(feature, "organization", None) or feature.project.organization
+    def has(self, feature: Union[Feature], actor: Any) -> bool:
+        organization = getattr(feature, "organization", None) or feature.project.organization  # type: ignore
```

Comment:
Why the ignore here?

Reply:
I was trying to get around the `Feature` not having `.project` but I solved this with `.get_organization()`.